plugin/evm/export_tx.go

```diff
@@ -356,6 +356,119 @@ func (vm *VM) newExportTx(
 	return tx, utx.Verify(vm.ctx, vm.currentRules())
 }
 
+// newUnsignedExportTX returns a new ExportTx without signing it.
```

Comment:
New method. It creates a single export TX using a `common.Address` directly, instead of deriving it from keys. Additionally, the resulting transaction is not signed, as my understanding is the signature is not necessary in the context of adjusting shared memory from a precompile.